app/services/github_checks_verifier.rb

```diff
@@ -11,6 +11,11 @@ class GithubChecksVerifier < ApplicationService
   config_accessor :check_name, :workflow_name, :client, :repo, :ref
   config_accessor(:wait) { 30 } # set a default
   config_accessor(:check_regexp) { "" }
+  config_accessor(:allowed_conclusions) { "success,skipped" }
+
+  def initialize
+    @allowed_conclusions_array = allowed_conclusions.split(/\s*,\s*/)
+  end
```

Comment:
I'd rather go for parsing the arguments in the entrypoint.rb, adding a line `allowed_conclusions = allowed_conclusions.split(',').map(&:strip)` just before the config block.
This way we don't need to add a new instance variable (@allowed_conclusions_array) which can be confusing.

Reply:
I agree

Reply:
Ok cool! Updated. Left a comment on one thing I'm not sure about. Tests pass though!